src/components/Swap/Summary/Summary.tsx

```diff
@@ -1,24 +1,30 @@
 import { useLingui } from '@lingui/react'
 import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { PriceImpact } from 'hooks/useUSDCPriceImpact'
-import { ArrowRight } from 'icons'
+import { ArrowDown, ArrowRight } from 'icons'
 import { PropsWithChildren } from 'react'
+import styled from 'styled-components/macro'
 import { ThemedText } from 'theme'
 import { formatCurrencyAmount } from 'utils/formatCurrencyAmount'
 
-import Column from '../../Column'
+import { default as Col } from '../../Column'
 import Row from '../../Row'
 import TokenImg from '../../TokenImg'
 
+const Column = styled(Col)<{ collapsedDetails: boolean }>`
+  justify-items: ${({ collapsedDetails }) => (collapsedDetails ? 'center' : 'left')};
+`
+
 interface TokenValueProps {
   input: CurrencyAmount<Currency>
   usdc?: CurrencyAmount<Currency>
+  collapsedDetails: boolean
 }
 
-function TokenValue({ input, usdc, children }: PropsWithChildren<TokenValueProps>) {
+function TokenValue({ input, usdc, collapsedDetails, children }: PropsWithChildren<TokenValueProps>) {
```

Comment:
`open` is idiomatically used for expandos, so feel free to replace `collapsedDetails` with `open`.